Jenkinsfile

```diff
@@ -36,6 +34,8 @@ timestamps {
         userRemoteConfigs: scm.userRemoteConfigs
       ])
 
+      sh 'git clean -dffx'
```

Comment:
this git clean done manually may mess up the state of the workspace and you may not be building the correct commit as a result...